membership-attribute-service/app/services/GuardianPatronService.scala

```diff
@@ -73,14 +73,11 @@ class GuardianPatronService(
       deliveryAddress = None,
       subscription = Subscription(
         id = Id(subscription.id),
-        name = Name(subscription.id),
+        subscriptionNumber = Name(subscription.id),
```

Comment:
should rename the value class to match as well
```suggestion
        subscriptionNumber = SubscriptionNumber(subscription.id),
```

Reply:
done this in the second commit on this PR